packages/jazz-tools/src/media/create-image.test.ts

```diff
@@ -10,7 +10,7 @@ describe("createImage", async () => {
   const getPlaceholderBase64 = vi.fn();
   const createFileStreamFromSource = vi
     .fn()
-    .mockResolvedValue(FileStream.create({ owner: account.$jazz.owner }));
+    .mockResolvedValue(FileStream.create({ owner: account }));
```

Comment:
There is actually a change in semantics here: `account.$jazz.owner` returns undefined now, so the original code would create a brand new group. This could lead to hard-to-debug issues when updating. Maybe we shouldn't allow owner to be undefined?

Reply:
If we're going to throw, I'd rather just prevent `owner` from being undefined at the type-level